api/src/Entity/User.php

```diff
@@ -233,7 +236,9 @@ public function getRoles(): array
         // guarantee every user at least has ROLE_USER
         $roles[] = 'ROLE_USER';
 
-        return array_values(array_unique($roles));
+        return $this->roles;
+
+        return array_unique($roles);
```

Comment:
```suggestion
        return $this->roles;

        return array_unique($roles);
```
Warning you have 2 return, You don't keep `return array_values(array_unique(roles));`?